style.css

```diff
@@ -1,3 +1,92 @@
 * {
   font-family: sans-serif;
 }
+
```

Comment:
- [ ] Kindly try to stick with the wireframe as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md#:~:text=Add%20CSS%20styles%20to%20the%20application%20to%20make%20it%20match%20this%20wireframe%3A)

| Your design | Wireframe | 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/176219579-3820d07e-83fc-4ba2-8c4b-6630ebe7dd6a.png) | ![image](https://user-images.githubusercontent.com/24703171/176219707-d0b736f8-915f-4da6-a881-568f47c0a894.png) |

You can use the styles of your choice (colors, fonts, backgrounds, etc.), but try to follow the general layout.

Reply:
fixed